src/main/scala/scalalab3/chatbotengine/telegram/PoolingEngineActor.scala

```diff
@@ -0,0 +1,60 @@
+package scalalab3.chatbotengine.telegram
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.pattern.pipe
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.duration.Duration
+
+import scalalab3.chatbotengine.core.ChatBot
+import scalalab3.chatbotengine.model.{OutMessage, Update}
+
+object PoolingEngineActor {
+  def props(bots: Traversable[ChatBot]): Props = Props(new PoolingEngineActor(bots))
+
+  case class Tick(ref: ActorRef)
+  case class BatchUpdate(dest: ActorRef, payload: Seq[Update])
+}
+
+class PoolingEngineActor(bots: Traversable[ChatBot]) extends Actor with ActorLogging {
+  import PoolingEngineActor._
+  import context.dispatcher
+
+  val registry = bots.map { b =>
```

Comment:
there is code conventions for public interface  http://docs.scala-lang.org/style/types.html
Please use explicit type declaration here